entity-framework/core/extensions/index.md

```diff
@@ -17,31 +17,25 @@ These tools and extensions provide additional functionality for Entity Framework
 
 ### LLBLGen Pro
 
-LLBLGen Pro is an entity modeling solution with support for Entity Framework and Entity Framework Core. It lets you easily define your entity model and map it to your database, using database first or model first, so you can get started writing queries right away. For EF Core: 2, 3, 5, 6.
```

Comment:
Extensions/tools may choose to support EF versions which are out-of-support - I think it's OK to continue listing those versions.

Reply:
I recently removed extensions that only supported v 2. Maybe do the same (later) for the ones that do not support 6?

Reply:
Yeah, it's a question of how much time we want to wait... 2 is really old by now, whereas 3.1 is just now going out of support - so we may want to give it a while.

Reply:
Sure, wait until past release of 8 of course.